### DISCUSSION TITLE
Nicknamer issue

### DISCUSSION THREAD

**Discussion by Misquoth** [TownyAdvanced/Towny#4836] [Q&A]
I'm using Nicknamer and It seems like it does change the name for command purposes and others can see the new name, but for myself (and dynmap) only the original name shows.

Looking for help on how to troubleshoot this and if it is potentially a bug.

Thanks.

**Comment by LlmDl** [comment]
I dont see how this relates to Towny.

**Reply by Misquoth** [reply]
I'm thinking that Towny chat is overriding it. 

However I will try and clone my install, remove towny and see if the issue remains.

**Reply by LlmDl** [reply]
File a bug report ticket instead of opening a discussion if you find something.
